### PR TITLE
feat: add storybook

### PR DESCRIPTION
**Pull request by Maith8Ae**
This PR introduces [storybook](https://storybook.js.org/) into this project which makes it easy to tweak UI components complying a unified design language. Our design guide here https://scene.zeplin.io/project/6006d3fa682b2d89770dae5d.

### CODE REVIEW

**Comment by maybeast** [comment]
Think we can get this in once the build is passing @stackingcats?

This would be helpful for gradually implementing the new design.

**Comment by scorebreaker** [comment]
The build is failing, because prettier wasn't run before the commit. I am not familiar with the storybook itself, but I tested it and it looks to be working.

**Comment by stackingcats** [comment]
- Needs a rebase
- Needs `yarn prettier`

Then it's good to go :+1:

**Comment by stackingcats** [comment]
Rebased and solved the prettier issue. Also added a couple of basic Button stories. I think we can merge this and start adding another components on the go.

File: .storybook/theme.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { create } from "@storybook/theming";
+import { darkTheme } from "../src/themes";
+
+export default create({
+  base: "dark",
+  brandTitle: "OpenDEX storybook",
+  brandImage:
+    "https://gblobscdn.gitbook.com/spaces%2F-Lq7dAvRcXVJXcPrZrNF%2Favatar.png?alt=media",
```
Comment (maybeast) [inline .storybook/theme.js:8]: This image might stop working as new version is pushed and CDN cache is invalidated. Might need to add a more permanent link to OpenDEX logo image.

**Review by maybeast — APPROVED** [review]

**Comment by maybeast** [comment]
Excellent work. Thanks @Maith8Ae for starting and @stackingcats for finishing it.

**Pull request merged**